src/enki.py

```diff
@@ -28,42 +23,52 @@
     if not os.path.exists(item):
         print(f"\nENKI ERROR: '{item}' doesn't exist in your repository.")
         args.path.remove(item)
-        continue
+        sys.exit(2)
 
 user_input = args.path
 
 
-if args.command == 'generate':
+def expand_file_paths(item):
+    """Expand filepaths."""
+    expanded_files = []
+
+    for dirpath, dirnames, filenames in os.walk(str(item) + '/'):
+        for name in filenames:
+            if name.startswith('_') or not name.endswith('adoc') or name == 'master.adoc' or name == 'README.adoc':
```

Comment:
This might also be a minor nitpick, but if you flipped the condition around, you wouldn't need the else clause or an explicit `continue` call.